code/obj/artifacts/artifact_items/pitcher.dm

```diff
@@ -0,0 +1,193 @@
+/obj/item/reagent_containers/food/drinks/drinkingglass/artifact
+	name = "artifact pitcher"
+	icon = 'icons/obj/artifacts/artifactsitem.dmi'
+	desc = "You have no idea what this thing is!"
+	artifact = 1
+	module_research_no_diminish = 1
+	mat_changename = 0
+	can_recycle = 0
+
+	New(var/loc, var/forceartitype)
+		..()
+		var/datum/artifact/pitcher/AS = new /datum/artifact/pitcher(src)
+		if (forceartitype)
+			AS.validtypes = list("[forceartitype]")
```

Comment:
```suggestion
	New(var/loc, var/forceartiorigin)
		..()
		var/datum/artifact/pitcher/AS = new /datum/artifact/pitcher(src)
		if (forceartiorigin)
			AS.validtypes = list("[forceartiorigin]")
```
to make it also use the new name for the var that I changed in my patch that just got merged ^_^

Reply:
woops, didn't see that. Will add in a sec